code/obj/cement.dm

```diff
@@ -6,7 +6,9 @@
 	density = 0
 	layer = OBJ_LAYER + 0.9
 	event_handler_flags = USE_CANPASS
-	var/health = 4 //how many hits to destroy + 1
+	var/const/initial_health = 4
```

Comment:
why is one baseHealth and the other one initial_health

Reply:
Just wanted to reuse the number instead of it being changed in one place and not being updated in another (code wise)